src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -35,6 +35,7 @@ const createAuthenticatorOptions = (options = []) => {
     return {
       label: option.label,
       value: valueObject,
+      relatesTo: option.relatesTo,
       authenticatorType: option.relatesTo && option.relatesTo.type,
```

Comment:
I didn't want to remove `authenticatorType` itself as it seems like it's used in many places. I thought it would be an overkill for this PR.

Reply:
what else beside used in `FactorUtil.getFactorData`?

Reply:
I just did a global search and there seems to be many occurrences. However, another point to note - if I change getFactorData to accept an object instead, it will need to change the way it's used in `BaseAuthenticatorView` as well.

Reply:
- global search doesn't give you correct insights where this particular authenticatorType has been used.
- we probably dont want to change `getFactorData` but `getFactorData(relates.value.authenticatorType)`
- may also making same change to another function for create enroll authenticator.

you can leave as separate ticket when you have time to dig into more details.

Reply:
Opened this - https://oktainc.atlassian.net/browse/OKTA-315716